src/test/librbd/mock/MockImageCtx.h

```diff
@@ -56,6 +57,7 @@ struct MockImageCtx {
 
   ~MockImageCtx() {
     wait_for_async_requests();
+    image_ctx->op_work_queue->drain();
```

Comment:
Not currently needed for the mock tests -- they have an explicit redirector that punts calls to ```MockImageCtx::op_work_queue::queue``` to ```ImageCtx::op_work_queue::queue```.  See ```expect_op_work_queue```.